### PR TITLE
User Playlists

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: src/containers/account/playlists.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+			// TODO: Only invalidate `useGetAccountPlaylistsPageDataQuery`
+			await queryClient.invalidateQueries();
+		},
+	});
+
+	async function addPlaylist(e: FormEvent<HTMLFormElement>) {
+		e.preventDefault();
+
+		if (!title) {
+			setErrors(['missing title']);
```
Comment (mattleff) [inline src/containers/account/playlists.tsx:33]: i18n here and below

File: src/containers/account/profile.tsx [hunk 2]
```diff
@@ -1,25 +1,135 @@
...
-				value={givenName}
-				onChange={(e) => setGivenName(e.target.value)}
+		<form onSubmit={submit}>
+			<ul>
+				{errors.map((e) => (
+					<li key={e}>{e}</li>
+				))}
+			</ul>
+
+			<Input label={'email'} type={'email'} value={email} setValue={setEmail} />
```
Comment (mattleff) [inline src/containers/account/profile.tsx:97]: i18n for all these labels

File: src/containers/account/profile.tsx [hunk 3]
```diff
@@ -1,25 +1,135 @@
...
-	// TODO: make sure givenName doesn't get populated with email on login (why??)
-	// TODO: fix err: "A component is changing an uncontrolled input of type text to be controlled."
 	return (
-		<form>
-			<input
-				type={'text'}
-				value={givenName}
-				onChange={(e) => setGivenName(e.target.value)}
+		<form onSubmit={submit}>
+			<ul>
```
Comment (mattleff) [inline src/containers/account/profile.tsx:91]: How about a `<FormErrors>` component that would take an array of errors and display it in a standard way?

**Review by mattleff — APPROVED** [review]
Looks great! 🍓

File: src/containers/account/profile.tsx (hunk 3 above)
Comment (narthur) [reply src/containers/account/profile.tsx:91]: Punting until it's clearer what direction we're going for error handling.

**Pull request merged**